assets/src/edit-story/app/media/utils/getFirstFrameOfVideo.js

```diff
@@ -26,7 +26,7 @@ function getFirstFrameOfVideo(src) {
   const video = document.createElement('video');
   video.muted = true;
   video.crossOrigin = 'anonymous';
-  video.preload = 'metadata';
+  video.preload = 'auto';
```

Comment:
A comment would be nice explaining this and why it's needed.

Reply:
Will add Chrome note, but giving it a fresh look it should be self-explanatory:
We want to get the first frame of the video and yet we preload only the metadata (not the actual content) - so it should not work, just some browsers (Chrome) behavior is misleading because it also preloads the content.

Reply:
Are you going to add a comment?

Reply:
Yes.